docs/sources/enterprise/saml.md

```diff
@@ -109,6 +117,55 @@ Grafana provides configuration options that let you modify which keys to look at
 
 An example is `assertion_attribute_name = "givenName"` where Grafana looks within the assertion for an attribute with a friendly name or name of `givenName`. Both, the friendly name (e.g. `givenName`) or the name (e.g. `urn:oid:2.5.4.42`) can be used interchangeably as the value for the configuration option.
 
+### Configure team sync
+
+> Only available in Grafana v7.0+
```

Comment:
This is misleading. Team Synch was available before 7.0. Please clarify what functionality this refers to.

Reply:
This refers to SAML only. Probably,
> Team sync support for SAML only available in Grafana v7.0+

Reply:
SAML did not support Team Sync before 7.0 so this is accurate. Perhaps we can make that more clear somehow?

Reply:
I like @alexanderzobnin 's suggestion. Let's go with that.

Reply:
Looks like this still needs updating.